packages/local_auth/local_auth_darwin/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 60;
+	objectVersion = 54;
```

Comment:
I'm not sure why this was changed. However, this seems reaosonable

Reply:
IIRC this is just based on what version of Xcode was last used to write the project file; it shouldn't matter.